AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/InjectExtraTest.java

```diff
@@ -22,81 +22,113 @@
 import java.util.List;
 
 import org.androidannotations.test.parceler.ParcelerBean;
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.Robolectric;
 import org.robolectric.RobolectricTestRunner;
-import org.robolectric.util.ActivityController;
+import org.robolectric.android.controller.ActivityController;
 
 import android.app.Activity;
 import android.content.Context;
 
 @RunWith(RobolectricTestRunner.class)
 public class InjectExtraTest {
 
-	private ExtraInjectedActivity_ activity;
 	private Context context = Robolectric.buildActivity(Activity.class).create().get();
-	private ActivityController<ExtraInjectedActivity_> controller;
-
-	@Before
-	public void setUp() {
-		controller = Robolectric.buildActivity(ExtraInjectedActivity_.class);
-		activity = controller.get();
-	}
 
 	@Test
 	public void simpleStringExtraInjected() {
-		controller.withIntent(ExtraInjectedActivity_.intent(context).stringExtra("Hello!").get()).create();
+		ExtraInjectedActivity activity = Robolectric.buildActivity(ExtraInjectedActivity_.class, ExtraInjectedActivity_.intent(context)
```

Comment:
MAYBE extract the intent into an own variable to make the code a bit better readable?